cli/README.md

```diff
@@ -43,14 +43,13 @@ keptn [command] [entity] [name] [flags]
 
 where **command**, **entity**, **name**, and **flags** are:
 
-- **command**: Specifies the operation that you want to perform, for example, install, create, onboard, send.
+- **command**: Specifies the operation that you want to perform, for example, install, create, send.

```

Comment:
```suggestion
- **command**: Specifies the operation that you want to perform, such as install, create, or send.

```

Reply:
Thanks! I've made the adjustment here: 47bac16388b6e5aaaf846b536eb47cddc7a090f2